meta-ov/recipes-core/psplash/files/rotation

```diff
@@ -0,0 +1 @@
+ROTATION=0
```

Comment:
Hm, we have this rotation in bunch of files already. I wonder if we can use info from /boot or /dev (don't have exact paths at hand at the moment, but I can look up if necessary.